### PR TITLE
W3ECO-38 Home screen markup

### PR DESCRIPTION
**Pull request by Max-Levitskiy**
W3ECO-54 Second round gameplay screen markup

### CODE REVIEW

File: public/game/bubble.png
Comment (SteffQing) [inline public/game/bubble.png:1]: This seems to be an issue on the bubble image itself. The floating width and height of the bubble some times encompasses another bubble, allowing me to click on bubble, but pop more than that 
That's a design flaw

File: src/components/GameCard.tsx
Comment (SteffQing) [inline src/components/GameCard.tsx:1]: Endpoint ready. We can remove the static LEVELS_SCORE variable in next commit

File: src/components/Menu.tsx
Comment (SteffQing) [inline src/components/Menu.tsx:1]: fill path for svg in the Menu icons are not functional yet

File: src/components/game/tree/FlyingItem.tsx
Comment (SteffQing) [inline src/components/game/tree/FlyingItem.tsx:1]: My last update of this file utilized the img src directly, by checking the boolean value of isBug, it determined what image resource to display
I think that's better, no?

File: src/assets/svg/Level.tsx
Comment (SteffQing) [inline src/assets/svg/Level.tsx:1]: Are we deleting this?

**Review by SteffQing — COMMENTED** [review]
I will test the app live

File: vite.config.ts
Comment (SteffQing) [inline vite.config.ts:1]: Duplicate "resolve" object

File: src/components/Timer.tsx [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
-import { FlexBoxCol } from './styled/styled';
+import React, { useState, useEffect } from "react";
+import { FlexBoxCol } from "./styled/styled";
 
-export function Timer(props: {initialSeconds: number, setIsGameOver: React.Dispatch<React.SetStateAction<boolean>>}) {
-    const [seconds, setSeconds] = useState(props.initialSeconds);
+export function Timer(props: {
+  initialSeconds: number;
+  setIsGameOver: React.Dispatch<React.SetStateAction<boolean>>;
+  flag?: boolean;
```
Comment (prokopofffff) [inline src/components/Timer.tsx:7]: i think we should give more informative name

File: src/components/game/ui/StartGameButton.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export function StartGameButton(props: Props) {
     setTime(_time);
     if (_time > 0) setIsGameReady(false);
   }, [nextGame]);
-
+  console.log(nextGame, time);
```
Comment (prokopofffff) [inline src/components/game/ui/StartGameButton.tsx:33]: remove logs

File: src/components/ui/Background.tsx [hunk 3]
```diff
@@ -2,34 +2,38 @@ import { cn } from "@/lib/tailwind-classes";
...
           positions[position],
+          colors[color]
         )}
       ></div>
-      {/*{main ? (*/}
-      {/*  <div className="absolute w-[350px] h-[350px] blur-[20px] rounded-full bg-[radial-gradient(circle_at_center,_rgba(160,_236,_25,_1),_rgba(160,_236,_25,_0.1))] top-[442px] -left-12 opacity-20"></div>*/}
-      {/*) : (*/}
-      {/*  <div className="absolute w-[350px] h-[350px] blur-[20px] rounded-full bg-[radial-gradient(circle_at_center,_rgba(25,_160,_236,_1),_rgba(25,_160,_236,_0.1))] opacity-20"></div>*/}
-      {/*)}*/}
+      {/* {true ? (
```
Comment (prokopofffff) [inline src/components/ui/Background.tsx:32]: remove comments

File: src/pages/Main.tsx [hunk 4]
```diff
@@ -21,8 +21,9 @@ export function Main() {
     setUsername(window.Telegram.WebApp.initDataUnsafe.user.username);
     getGameStatus()
       .then((res) => {
+        console.log(res);
         setRemainedGames(res.remainedGames);
-        if (res.remainedGames == 0) {
+        if (res.remainedGames <= 3) {
```
Comment (prokopofffff) [inline src/pages/Main.tsx:26]: less than 3, not equal

File: src/components/Timer.tsx (hunk 1 above)
Comment (SteffQing) [reply src/components/Timer.tsx:7]: Okay

File: src/components/game/ui/StartGameButton.tsx (hunk 2 above)
Comment (SteffQing) [reply src/components/game/ui/StartGameButton.tsx:33]: Sorry about that

File: src/components/ui/Background.tsx (hunk 3 above)
Comment (SteffQing) [reply src/components/ui/Background.tsx:32]: Got it

File: src/pages/Main.tsx (hunk 4 above)
Comment (SteffQing) [reply src/pages/Main.tsx:26]: Yeah, that was when I was testing
Forgot to update the logic
Thanks for pointing that out

**Pull request closed without merging**